### DISCUSSION TITLE
Cannot start nextpyp service

### DISCUSSION THREAD

**Discussion by monsekey** [nextpyp/.github#130] [Q&A]
Hi, I'm trying to deploy the nextpyp on my Rocky Linux 8.1 system. Tried both my personal sudo account and root account, under the folder "/home/myaccount/nextpyp", both can download the file, but "systemctl start nextpyp" always returns "ExecStart=/usr/bin/nextpyp-start (code=exited, status=203/EXEC)".
I know 203 is a permission error, I tried to run the following GPT suggested commands to gain the permission, under the root account:

SELinux is preventing /usr/lib/systemd/systemd from execute access on the file /usr/bin/nextpyp-start. For complete SELinux messages run: sealert -l 0eb7ce18-9d2e-422f-91c0-159d5156f1e0
ausearch -c '(yp-start)' --raw | audit2allow -M my-ypstart
semodule -X 300 -i my-ypstart.pp
systemctl start nextpyp

But still got the error mentioned at the beginning.

Also, I realized when I tried to executive the uninstall to restart, it runs out of permission, even if I chmod added, the tutorial folder requires root to remove it.

I wonder if someone can answer if these failures (primarily installation) are caused by the CentOS, the path, or strict restrictions set by the system manager.

Thanks!

**Comment by cuchaz** [comment]
Hi there and thanks for reporting the issue. After looking into the installer script, I think I may have found a bug in the script that caused the wrong user account to be used for some operations. This may have led to permissions errors, like the one you experienced.

I've updated the installer script at `https://nextpyp.app/files/pyp/latest/install` to fix the error. If you try re-downloading the installer and re-installing nextpyp using the new script, hopefully things will work better next time?
